gatsby/content/projects/2017-05-02-thunderbird.mdx

```diff
@@ -3,14 +3,34 @@ layout: projectimage
 title: Mozilla Thunderbird
 categories: 
  - client
-description: A free email & chat app
-author: Frédéric Wang
+description: A free email, calendar & chat app
+author: Mozilla
 maturity: Beta
 language: C
```

Comment:
The source code doesn't look like C - rather, JavaScript?

Reply:
Well, depends on what part of the source, the Matrix specific part is indeed mostly implemented in JS

Reply:
This really should list half a dozen languages, but that doesn't seem possible. I think by core LOC (which is a useless measure, but whatever) "C" is what most of Thunderbird is. (If you exclude all of Gecko, which is part of Firefox, maybe this isn't true...)

I wonder if we can/should just leave this field blank.

Reply:
I’m guessing the point of this field is “if I wanted to contribute to *this*, which language would I need to know/learn”, where this is “this matrix related thing”. In this context, as I understand it, JavaScript only is relevant.

Reply:
Ah, yeah from the POV of a contributor having this field makes sense. From the POV of a user, not so much!